lib/constants.js

```diff
@@ -10,6 +10,7 @@ const WDA_RUNNER_BUNDLE_ID = 'com.facebook.WebDriverAgentRunner';
 const WDA_RUNNER_APP = 'WebDriverAgentRunner-Runner.app';
 const WDA_SCHEME = 'WebDriverAgentRunner';
 const PROJECT_FILE = 'project.pbxproj';
+const WDA_BASE_URL = 'http://localhost';
```

Comment:
I think it makes sense to change localhost to 127.0.0.1, since sometimes it can be resolved to ::1, which is not supported by iproxy